packages/web/components/Hero/styles.js

```diff
@@ -1,7 +1,13 @@
 import styled from 'styled-components';
 
+const heroImage = () => {
+	const heroImgs = ['1.jpg', '2.jpg', '3.jpg', '4.jpg', '5.jpg', '6.jpg', '7.jpg'];
+	const heroIndexImg = Math.floor(Math.random() * heroImgs.length);
```

Comment:
```suggestion
	const heroIndexImg = Math.floor(Math.random() * heroImgs.length + 1);
```

Precisa somar o length do array + 1, se não você estará excluindo o último índice

Reply:
@luizeboli mas nesse caso o total ali seria 8 e não 7 (que é o valor do length do array). Assim ele não iria ficar tentando acessar o índice 8 do array? Cê não ta confundindo por conta do nome das imagens? hehehe

Reply:
@davioliveiira não, pois pensa no seguinte, o máximo que o `Math.random` retorna é 0.9999..

Agora, 0.999 * 8 é igual a 7.992..

O `Math.floor` sempre arredonda pro inteiro da casa anterior, ou seja, 7.99 vai virar 7.

Se você multiplicar apenas pela length do array, subtrai 1 dessa conta que eu fiz: 0.999 * 7 = 6.999, no floor vira 6. Assim, o último índice sempre é excluído

Reply:
Aah, agora entendi @luizeboli ! Alterei